src/browser/monkey-menu.js

```diff
@@ -207,6 +229,21 @@ function onLoad(event) {
       });
     }
   }
+
+  // TODO: Remove when file picker bugs are fixed
+  // https://bugzilla.mozilla.org/show_bug.cgi?id=1366330
+  // document.getElementById('import-script-file')
+  //     .addEventListener('change', onImportUserScript);
+  function preventAll(event) {
+    event.preventDefault();
+    event.stopPropagation();
+    event.stopImmediatePropagation();
+  }
+  document.getElementById('import-script-file')
```

Comment:
The element with this ID is hidden by CSS, so is not clickable, so this code doesn't work.

Reply:
It is indeed clickable. Otherwise the installation page would never open.

Reply:
When I check out this branch I see:

![untitled](https://user-images.githubusercontent.com/84729/35751172-8880b016-0825-11e8-8c1d-ba27b67636c9.png)

The \<a href="#import-user-script" ...\> and its text, but no \<input\>.  Clicking the (visible) anchor does nothing.  The CSS sets opacity to zero so maybe it's supposed to be overlaid (towards what end?), but that's broken.  If remove the opacity setting:

![untitled1](https://user-images.githubusercontent.com/84729/35751376-41360412-0826-11e8-91a5-8a7208daac8c.png)

The position is off?  Like I said above, the half-this-half-that implementation is suboptimal.

Reply:
Ah, must have to do with the recent fixes to the MM styling. When I tested it just happen to be in the right location to work.

> (towards what end?),

I thought this was obvious. You can't style input fields. So either you hide it and overlay it on text you would like to display or you're stuck with a "browse" button with absolutely no context. Now, I can grant you that at this moment in time the file picker is unnecessary, since they don't work (RIP Mozilla please). But down the line, unless you want to stick with the poor UI of a file input, the overlay is pretty much required.

Either way, I'll remove the boilerplate as it's, as you said, not needed at the time.